src/renderer/components/+apps-helm-charts/helm-chart-details.tsx

```diff
@@ -26,10 +25,11 @@ export class HelmChartDetails extends Component<Props> {
   @observable readme: string = null;
   @observable error: string = null;
 
-  private chartPromise: CancelablePromise<{ readme: string; versions: HelmChart[] }>;
+  private chartPromise: Promise<{ readme: string; versions: HelmChart[] }>;
```

Comment:
Not sure how these changes are related to the PR title?

Reply:
Specifically for this, see note in PR description

> Remove CancelablePromise as it was only used in one place and the return types where wrong causing a crash while debugging the above.

If you want I can move the out to a separate PR but I did encounter an exception while debugging this issue.

Reply:
Ok, missed the description. Probably fine to keep here as it's somewhat related.